LibertyDSNP/example-client
# converted all js files in src to ts with types

Purpose
---------------
We want to have the entirety of our src written in typescript but some js files slipped through without being converted.


Solution
---------------
Convert files in src from js to ts.


Change summary
---------------
* Converted and added types for torus wallet files
* Converted session storage file and added types.


Steps to Verify
----------------
1. Check all files in src
2. No js files should exist
3. Example client should still run
